src/app/lib/globalStyles/index.jsx

```diff
@@ -22,9 +22,10 @@ const GlobalStyleComponent = createGlobalStyle`
     ${({ fonts }) => fonts && createFontStyles(fonts)}
   `;
 
-const GlobalStyle = () => {
-  const { fonts } = useContext(ServiceContext);
-  return <GlobalStyleComponent fonts={fonts} />;
+// eslint-disable-next-line react/prop-types
+const GlobalStyle = ({ fonts }) => {
+  const styleFonts = useContext(ServiceContext).fonts;
```

Comment:
```suggestion
  const { fonts: styleFonts } = useContext(ServiceContext);
```

Reply:
Also why didn't we implement @dr3's suggestion here? https://github.com/bbc/simorgh/pull/2425#discussion_r304580081

Reply:
This will result to a failuire due to this ```fonts.reduce((fontStyles, fontStyle) => fontStyles + fontStyle, '')```